src/components/TopBar/TopBar.js

```diff
@@ -236,6 +248,12 @@ const TopBar = ({
         }
     };
 
+    const changeLanguage = (event, value) => {
+        if (onLanguageClick && value !== null) {
```

Comment:
onLanguageClick !== null 
value couldn't be null.

Reply:
With ToggleButton API, if you click on the button that is already selected, value is null. Thus you can avoid to trigger the function.